lldp_8021qaz_cmds.c

```diff
@@ -655,9 +655,9 @@ _set_arg_tcbw(struct cmd *cmd, char *args, const char *arg_value,
 		err = cmd_invalid;
 		goto invalid;
 	} else if (test) {
+		free(parse);
```

Comment:
The last use of `parse` or `strtok()` in this function seems to be the `strtok()` on line 649 inside a `for` loop. This `free()` could be moved to just after the loop removing the need to free it later here and at the end of this function.
```suggestion
    for (i = 0; i < 8 && toked_bw; i++) {
		percent[i] = atoi(toked_bw);
		toked_bw = strtok(NULL, ",");
	}
	free(parse);
```